app/src/components/side-bar/settings/OptionsList.tsx

```diff
@@ -33,11 +39,15 @@ function OptionsList({ rows }: { rows: SideBarRowProps[] }) {
             <SideBarRow
               {...item}
               key={
-                item.privacyStatus !== undefined
-                  ? statusMap.privacy[item.privacyStatus].id
-                  : item.activityStatus !== undefined
-                    ? statusMap.activity[item.activityStatus].id
-                    : index
+                item.status !== undefined && item.type !== undefined
+                  ? item.type === StatusType.PRIVACY
+                    ? statusMap.privacy[item.status as privacySettingsID].id
+                    : item.type === StatusType.ACTIVITY
+                      ? statusMap.activity[item.status as activitySettingsID].id
+                      : statusMap.permission[
+                          item.status as permissionSettingsID
+                        ].id
+                  : index
```

Comment:
I think this should better be extracted into a new function with if elses as the nested ternary are hard to read